test/parallel/test-tls-wrap-econnreset-pipe.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const tls = require('tls');
+const net = require('net');
+
+common.refreshTmpDir();
+
+const server = net.createServer((c) => {
+  c.end();
+}).listen(common.PIPE, common.mustCall(() => {
+  tls.connect({ path: common.PIPE })
+    .once('error', common.mustCall((e) => {
+      assert.strictEqual(e.code, 'ECONNRESET');
+      assert.strictEqual(e.path, common.PIPE);
+      assert.strictEqual(e.port, undefined);
+      assert.strictEqual(e.host, undefined);
```

Comment:
Add a assertion here: `assert.strictEqual(e.localAddress, undefined)` just like you did for `port` and `host`?